### ISSUE TITLE
[Feb 2025] Accessibility Audit: Bulk upload patient user flows

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8515]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the bulk patient upload user flow. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/upload-patients`

Use cases to test:
* successful bulk upload (inc success toast)
* bulk upload produces validation messages
* bulk upload produces error toast

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See tickets from previous audits #5781, #6331

**Comment by emyl3** [comment]
Findings:
- `/app/upload-patients` - validation errors + error toast [dev.simplereport.gov-Import-patients-from-spreadsheet-validation-2025-03-03.csv](https://github.com/user-attachments/files/19061432/dev.simplereport.gov-Import-patients-from-spreadsheet-validation-2025-03-03.csv)
- `/app/upload-patients` - success [dev.simplereport.gov-Import-patients-from-spreadsheet-success-2025-03-03.json](https://github.com/user-attachments/files/19061464/dev.simplereport.gov-Import-patients-from-spreadsheet-success-2025-03-03.json)


Issues created:
- https://github.com/CDCgov/prime-simplereport/issues/8572
